test/scenario/MaxBasketSize.test.ts

```diff
@@ -574,20 +580,21 @@ describe(`Max Basket Size - P${IMPLEMENTATION}`, () => {
       if (REPORT_GAS) {
         await snapshotGasCost(backingManager.claimRewards())
       } else {
-        await expectEvents(backingManager.claimRewards(), [
-          {
-            contract: backingManager,
-            name: 'RewardsClaimed',
-            args: [compToken.address, rewardAmount.mul(20)],
-            emitted: true,
-          },
-          {
-            contract: backingManager,
-            name: 'RewardsClaimed',
-            args: [aaveToken.address, rewardAmount],
-            emitted: true,
-          },
-        ])
+        await backingManager.claimRewards()
+        // await expectEvents(backingManager.claimRewards(), [
```

Comment:
I think these can be uncommented, and the contract changed to be the 4626 ERC20s

Reply:
so if we really want to test that the events get fired, we would want to iterate over all the tokens in the basket and do some logic to know whether to expect aave or comp as a reward. we would also need to use anyValue because only one of each rewarding token will actually end up firing an event with any value but 0.

given that we test these events in dozens of other tests, and that this test is really just a gas test, i think we can just call `claimRewards` without using `expectEvents`